assets/pages/contributions-landing/contributionsLandingStyles.scss

```diff
@@ -0,0 +1,42 @@
+// -----  gu-sass ----- //
+
+@import '../../stylesheets/gu-sass/reset-src';
+@import '../../stylesheets/gu-sass/breakpoints';
+@import '../../stylesheets/gu-sass/colours';
+@import '../../stylesheets/gu-sass/layout';
+@import '../../stylesheets/gu-sass/webfonts';
+@import '../../stylesheets/gu-sass/typography';
+@import '../../stylesheets/gu-sass/helpers';
+@import '../../stylesheets/gu-sass/accessibility';
+
+// ----- Shared Components ----- //
+
+@import '../../components/headers/countrySwitcherHeader/countrySwitcherHeader';
+@import '../../components/countryGroupSwitcher/countryGroupSwitcher';
+@import '../../components/selectInput/selectInput';
+@import '../../components/circlesIntroduction/circlesIntroduction';
+@import '../../components/highlights/highlights';
+@import '../../components/contribute/contribute';
+@import '../../components/pageSection/pageSection';
+@import '../../components/secure/secure';
+@import '../../components/inlinePaymentLogos/inlinePaymentLogos';
+@import '../../containerisableComponents/contributionSelection/contributionSelection';
+@import '../../components/radioToggle/radioToggle';
+@import '../../components/numberInput/numberInput';
+@import '../../components/errorMessage/errorMessage';
+@import '../../components/contributionPaymentCtas/contributionPaymentCtas';
+@import '../../components/legal/termsPrivacy/termsPrivacy';
+@import '../../components/footer/footer';
+@import '../../components/svgs/svg';
+@import '../../components/ctaLink/ctaLink';
+@import '../../components/legal/contribLegal/contribLegal';
+@import '../../containerisableComponents/payPalContributionButton/payPalContributionButton';
+
+// ----- Pages ----- //
+
+@import './contributionsLanding';
+
+// ----- Guardian Egyptian Web Fonts ----- //
+
+@import '../../stylesheets/garnett';
+
```

Comment:
As discussed offline, this could be put inside the `contributionsLanding.scss` file, and that could be used as the webpack entry point.